src/scan.c

```diff
@@ -210,22 +212,31 @@ const char *token_str(token_t tok)
    return "???";
 }
 
-static const char *get_cond_analysis_identifier(const char *name)
+void conda_id_init()
 {
-   if (strcmp(name, "VHDL_VERSION") == 0)
-      return standard_text(standard());
-   else if (strcmp(name, "TOOL_TYPE") == 0)
-      return "SIMULATION";
-   else if (strcmp(name, "TOOL_VENDOR") == 0)
-      return PACKAGE_URL;
-   else if (strcmp(name, "TOOL_NAME") == 0)
-      return PACKAGE_NAME;
-   else if (strcmp(name, "TOOL_EDITION") == 0)
-      return "";
-   else if (strcmp(name, "TOOL_VERSION") == 0)
-      return PACKAGE_VERSION;
-   else
-      return NULL;
+   cond_idents = shash_new(16);
+
+   conda_id_add("VHDL_VERSION", standard_text(standard()));
+   conda_id_add("TOOL_TYPE",    "SIMULATION");
+   conda_id_add("TOOL_VENDOR",  PACKAGE_URL);
+   conda_id_add("TOOL_NAME",    PACKAGE_NAME);
+   conda_id_add("TOOL_EDITION", "");
```

Comment:
Can you put a non-empty string here? I noticed one of the VHDL-2019 compliance tests is checking for it. I've no idea what `TOOL_EDITION` is supposed to be - maybe you could check Aldec or similar.

Reply:
Riviera Pro has "2022.04 Linux64" so maybe just make this one PACKAGE_VERSION too.